<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Folder Watcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelFolder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder to watch:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEditFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBrowse">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWatch">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>711</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEditConsole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>711</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelConsole">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Console:</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>350</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>350</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
